zone.go

```diff
@@ -1,11 +1,13 @@
 package main
 
 import (
+	"sort"
 	"strings"
 	"sync"
 
 	"github.com/miekg/dns"
 	"github.com/rcrowley/go-metrics"
+	glob "github.com/ryanuber/go-glob"
```

Comment:
Is this necessary? It implies we support globbing, not just wildcard ('*') labels.

Reply:
Well, it provides more functionality for practically zero cost, so it's possible to use `subdomain*.domain.com`

Reply:
> Do you have a use case for needing globs?